packages/main/src/Popup.js

```diff
@@ -80,6 +81,19 @@ const metadata = {
 			defaultValue: undefined,
 		},
 
+		/**
+		 * Defines the IDs of the elements that label the component.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.0.3
```

Comment:
since 1.0.0

Reply:
I assume 1.1.0, like in the RadioButton. Done